src/formatting_stack/linters/eastwood.clj

```diff
@@ -53,8 +45,7 @@
                             :filename            (if (string? uri-or-file-name)
                                                    uri-or-file-name
                                                    (-> ^File uri-or-file-name .getCanonicalPath)))))
-         (concat (impl/warnings->reports output)
-                 (impl/exceptions->reports @exceptions)))))
+         (concat (impl/exceptions->reports @exceptions)))))
```

Comment:
Redundant `concat`

Reply:
I don't think it is redundant, although it could be changed to `into`

Reply:
Not sure why one would want to concat/into a coll alone?

Reply:
The concat is part of a thread-last operation, so it get's two arguments (reports from eastwood and exceptions).

I'd prefer `into` over `concat` because of it's eager nature.

Reply:
whoopsie